src/wasm-lib/kcl/src/execution/cache.rs

```diff
@@ -355,7 +356,7 @@ firstSketch = startSketchOn('XY')
 // Remove the end face for the extrusion.
 shell(firstSketch, faces = ['end'], thickness = 0.25)"#;
 
-        let (program, ctx, _) = parse_execute(old).await.unwrap();
+        let (program, _, ctx, _) = parse_execute(old).await.unwrap();
```

Comment:
At this point, if there's gonna be 4 values returned from `parse_execute`, I think we may as well returned a struct with named fields.

Reply:
Going to land this as a follow-up since I might change these again in my current PR